lib/google_coordinates.rb

```diff
@@ -0,0 +1,25 @@
+require 'httparty'
+
+class GoogleCoordinates
+  def self.location_info(location)
```

Comment:
I flattened one level of nesting, but I don't really know a good way to reduce the branching beyond that. I could merge some scenarios (i.e. having different failures return the same error message), but that loses specificity. I can remove the explicit returns if you think that helps. Do you have any suggestions for simplifying here?